src/main/java/frc/robot/RobotContainer.java

```diff
@@ -118,7 +113,7 @@ public RobotContainer() {
       }
      configureButtonBindings();
      mDriveSubsystem.setDefaultCommand(mDriveCommand);
-     mElevatorSubystem.setDefaultCommand(new moveLift());
+     //mElevatorSubystem.setDefaultCommand(new moveLift());
```

Comment:
You most likely need this, prior comment was me being confused because the name was moveLiftUp